puzzles/models.py

```diff
@@ -27,12 +27,12 @@ class InvalidMetaPuzzleError(PuzzleModelError):
 
 
 class Puzzle(models.Model):
-    name = models.CharField(max_length=80, unique=True)
+    name = models.CharField(max_length=80)
     hunt = models.ForeignKey('hunts.Hunt', on_delete=models.CASCADE, related_name='puzzles')
     url = models.URLField(blank=True)
 
-    sheet = models.URLField(default='', unique=True)
-    channel = models.CharField(max_length=128, default='', unique=True)
+    sheet = models.URLField(default='', unique=True, null=True)
```

Comment:
Should we replace `default=''` with `blank=True` for readability?

Reply:
I think this question is covered in the comment thread below: https://github.com/cardinalitypuzzles/smallboard/pull/193/files#r529159104